pkg/meshctl/validation/checks/agent_parameters.go

```diff
@@ -7,27 +7,23 @@ import (
 	"github.com/rotisserie/eris"
 )
 
-type ServerParams struct {
-	RelayServerAddress string
-}
-
 type parameterCheck struct{}
 
-func NewServerParametersCheck() *parameterCheck {
+func NewAgentParametersCheck() *parameterCheck {
 	return &parameterCheck{}
 }
 
 func (p *parameterCheck) GetDescription() string {
-	return "Gloo Mesh server parameters are valid"
+	return "Gloo Mesh Server Parameters"
```

Comment:
Server is a bit confusing because it feels like we're talking about the networking pod
```suggestion
	return "Gloo Mesh Agent Relay Parameters"
```